README.md

```diff
@@ -40,12 +40,12 @@ Once all of these have come up, navigate to `localhost:8080` in your browser and
 
 ### Getting Started
 
-In order to run a local environment, you'll need to use the [PAWLS CLI](cli/readme.md) to download the PDFs and metadata you want to serve. The PDFs should be put in `skiff_files/apps/pawls`.
+In order to run a local environment, you'll need to use the [PAWLS CLI](cli/readme.md) to preprocess and assign the PDFs you want to serve. When using PDFs from semantic scholar, the CLI is also used to download the PDFs. The PDFs have to be put in a directory structure within `skiff_files/apps/pawls` (see [PAWLS CLI usage](cli/readme.md#usage) for details).
```

Comment:
```suggestion
In order to run a local environment, you'll need to use the [PAWLS CLI](cli/readme.md) to preprocess and assign the PDFs you want to serve. When using PDFs from semantic scholar, the CLI is also used to download the PDFs. The PDFs have to be put in the directory structure within `skiff_files/apps/pawls` (see [PAWLS CLI usage](cli/readme.md#usage) for details).  If that directory doesn't exist, you can create it like so:

    mkdir -p skiff_files/apps/pawls

The application code references the same path as an absolute one from the root of the filesystem, i.e. `/skiff_files/apps/pawls`.  That's because we mount the directory in this repository at that location within the container. If you're deploying PAWLS yourself you'll need to make sure `/skiff_files/apps/pawls` exists.
```

Let me know what you think of this.